saleor/product/templatetags/product_images.py

```diff
@@ -40,7 +40,7 @@ def get_thumbnail(instance, size, method='crop'):
                              extra={'instance': instance, 'size': size})
         else:
             return thumbnail.url
-    return static('images/product-image-placeholder.png')
+    return static('images/placeholder' + size + '.png')
```

Comment:
@krzysztofwolski Can you help make sure that we select an existing placeholder here? I'm ok with either raising an exception if exact match is missing (not sure how easy that is to detect) or with selecting the closest larger alternative.